packages/web-components/src/horizontalbarchart/horizontalbarchart.spec.ts

```diff
@@ -0,0 +1,130 @@
+
+import { test } from '@playwright/test';
+import { expect, fixtureURL } from '../helpers.tests.js';
+
+test.describe('horizontalbarchart - Basic', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto(fixtureURL('components-horizontalbarchart--basic'));
+    await page.waitForFunction(() => customElements.whenDefined('fluent-horizontalbarchart'));
+  });
+
+  test('Should render horizontalbarchart properly', async ({ page }) => {
+    const element = page.locator('fluent-horizontalbarchart');
```

Comment:
horizontal bar chart has 3 variants. Add individual tests for all variants separately.